core/test/app_grc.cpp

```diff
@@ -13,13 +13,45 @@
 
 namespace grg = gr;
 
+template<typename T>
+struct ArraySource : public grg::Block<ArraySource<T>> {
+    std::array<grg::PortOut<T>, 2> outA;
+    std::array<grg::PortOut<T>, 2> outB;
+};
+
+ENABLE_REFLECTION_FOR_TEMPLATE(ArraySource, outA, outB);
+
+template<typename T>
+struct ArraySink : public grg::Block<ArraySink<T>> {
+    std::array<grg::PortIn<T>, 2> inA;
+    std::array<grg::PortIn<T>, 2> inB;
+};
+
+ENABLE_REFLECTION_FOR_TEMPLATE(ArraySink, inA, inB);
+
 struct test_context {
```

Comment:
boy-scout-rule: if you could refactor and change `test_context` -> `TestContext`

Reply:
Done